From f3fdd07ad565a23d59c903a90f0a5ece99252bff Mon Sep 17 00:00:00 2001
From: Eric Li <lizhi.xmu@gmail.com>
Date: Thu, 17 Sep 2026 01:22:09 +0800
Subject: [PATCH] build: use the lms mpv fork (remote Blu-ray image support)

Point mpv at https://github.com/eli103/mpv (mpv master + the lms patch that lets
--bluray-device be a URL and streams the image through bd_open_stream).
---
 msg.txt            | 4 ++++
 packages/mpv.cmake | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)
 create mode 100644 msg.txt

diff --git a/msg.txt b/msg.txt
new file mode 100644
index 0000000..9f1548a
--- /dev/null
+++ b/msg.txt
@@ -0,0 +1,4 @@
+build: use the lms mpv fork (remote Blu-ray image support)
+
+Point mpv at https://github.com/eli103/mpv (mpv master + the lms patch that lets
+--bluray-device be a URL and streams the image through bd_open_stream).
diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index a2954d5..b274054 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -25,7 +25,7 @@ ExternalProject_Add(mpv
         subrandr
         libsixel
         curl
-    GIT_REPOSITORY https://github.com/mpv-player/mpv.git
+    GIT_REPOSITORY https://github.com/eli103/mpv.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""
-- 
2.45.1.windows.1

